pkg/blockstorage/azure/azuredisk.go

```diff
@@ -68,7 +68,7 @@ func (s *AdStorage) VolumeCreate(ctx context.Context, volume blockstorage.Volume
 	tags := blockstorage.SanitizeTags(blockstorage.KeyValueToMap(volume.Tags))
 	diskName := fmt.Sprintf(volumeNameFmt, uuid.NewV1().String())
 	diskProperties := &azcompute.DiskProperties{
-		DiskSizeGB: azto.Int32Ptr(int32(volume.Size)),
+		DiskSizeGB: azto.Int32Ptr(int32(blockstorage.SizeInG(volume.SizeInBytes))),
```

Comment:
Are you sure this is in GB and not GiB?  See https://azure.microsoft.com/en-us/pricing/details/managed-disks/ for example - they interchange GB and GiB but in their pricing table they use GiB